### DISCUSSION TITLE
Letting contributors post on the blog

### DISCUSSION THREAD

**Discussion by CBID2** [mdn/mdn-community#436] [Content]
Hi there! :) I noticed that certain members can write posts for the MDN Blog. It would be beneficial if contributors, especially those who frequent this repository, could contribute to the blog.

**Comment by bsmth** [comment]
Hi @CBID2 - sorry for the delay in getting back to you. That's a super idea, there are two invited authors who have written for us so far, at the moment the blog is not open to community contributions, but that might change in future. We'll keep you posted!
